### PR TITLE
Easy Module Installation for Dev Team

### LINKED ISSUE #852: Easy AI Module installation shell script

### PR DESCRIPTION
**Pull request by chandralegend**
Closes #852 

in the jaseci_ai_kit folder, you can do the following

1. `bash install.sh all`--> Installs all the models
2. `bash install.sh bart_sum stt` --> Modules separated by a space, will find respective packages and install relevant modules

### CODE REVIEW

File: jaseci_ai_kit/test.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ for file in $(find . -name "test_$1*.py"); do
     pytest --exitfirst -v $file
     if [ $? -ne 0 ]; then
         echo $file Failed
-        exit 1
-        break
+        # exit 1
+        # break
```
Comment (ypkang) [inline jaseci_ai_kit/test.sh]: unintentional change?

File: jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # **Text to Speech with Voice Cloning (`vc_tts`)**
 
+| :warning: WARNING          |
+|:---------------------------|
+| This modules is requires strict dependancies. therefore if you plan to use this with other modules, makesure to install this module in the beggining. Otherwise other modules will break.|
```
Comment (ypkang) [inline jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md]: There are multiple typos and grammatical errors in this sentence. I don't usually pick on these but this one has a little too many.

File: jaseci_ai_kit/install.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Comment (ypkang) [inline jaseci_ai_kit/install.sh:1]: Let's add some quick instructions  in the readme on how to use this script.

**Review by ypkang — CHANGES_REQUESTED** [review]

File: jaseci_ai_kit/test.sh (hunk 1 above)
Comment (chandralegend) [reply jaseci_ai_kit/test.sh]: yep unintentional.

Comment (chandralegend) [reply jaseci_ai_kit/test.sh]: I will fix

File: jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md (hunk 2 above)
Comment (chandralegend) [reply jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md]: I'll fix

File: jaseci_ai_kit/install.sh (hunk 3 above)
Comment (chandralegend) [reply jaseci_ai_kit/install.sh:1]: On it

**Review by ypkang — APPROVED** [review]

**Review by ypkang — APPROVED** [review]

**Pull request merged**